client/src/components/Button/Button.js

```diff
@@ -1,45 +1,46 @@
-import React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
-import CircularProgress from '@material-ui/core/CircularProgress';
-import green from '@material-ui/core/colors/green';
-import MDButton from '@material-ui/core/Button';
-import CheckIcon from '@material-ui/icons/Check';
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import green from "@material-ui/core/colors/green";
+import MDButton from "@material-ui/core/Button";
+import CheckIcon from "@material-ui/icons/Check";
```

Comment:
Use single-quotes.

Reply:
As we did not decide to use a double quote or single quote and also eslint is not forcing me to use a single quote so We can go with using both quotes.